### PR TITLE
f-user-message@0.3.0 Top margin removed

### PR DESCRIPTION
**Pull request by zlatinivanov-je**
_Removed component top margin. This should be set in the host app, if required_
_Issue #160  Updated dependencies and removed unused ones. Reduced bundle size_

Before: 
Screenshot 2020-07-16 at 17 02 25
After: 
Screenshot 2020-07-16 at 17 09 42

It pushes the page content down, below header, creating an unwanted gap. See screenshot below. 

---

## UI Review Checks

Screenshot 2020-07-16 at 15 32 12

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [x] Internet Explorer 11
- [x] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

**Review by DamianMullins — DISMISSED** [review]

File: packages/f-user-message/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-user-message",
   "description": "Fozzie User Message – Globalised User Message Component",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (ashleynolan) [inline packages/f-user-message/package.json]: Super NIT: this should be a minor version bump. Otherwise LGTM.

Comment (zlatinivanov-je) [reply packages/f-user-message/package.json]: Done :)

**Review by simonsandersje — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

**Review by ashleynolan — DISMISSED** [review]
LGTM

**Review by asherstoppard — DISMISSED** [review]

File: packages/f-user-message/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+Latest (roll into next release)
```
Comment (ashleynolan) [inline packages/f-user-message/CHANGELOG.md]: Can delete this line now

**Review by ashleynolan — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/f-user-message/package.json [hunk 3]
```diff
@@ -37,17 +37,16 @@
...
+    "axios": "0.19.2"
   },
   "peerDependencies": {
     "@justeat/eslint-config-fozzie": "3.3.1",
     "@justeat/f-trak": "0.6.0"
   },
   "devDependencies": {
     "@vue/cli-plugin-babel": "4.4.6",
-    "@vue/cli-plugin-eslint": "3.9.2",
+    "@vue/cli-plugin-eslint": "4.4.6",
```
Comment (ashleynolan) [inline packages/f-user-message/package.json]: I don't think you can upgrade this without all packages being upgraded too – that's why the build is failing atm

File: packages/f-user-message/CHANGELOG.md [hunk 4]
```diff
@@ -4,17 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+v0.3.0
 ------------------------------
 *July 23, 2020*
 
 ### Changed
 - Removed component top margin. This should be set in the host app, if required
 - Updated packages. Reduced bundle size
 
-v0.1.1
+v0.2.0
```
Comment (xander-marjoram) [inline packages/f-user-message/CHANGELOG.md:14]: Looks like you've squished v0.1.1?

Comment (zlatinivanov-je) [reply packages/f-user-message/CHANGELOG.md:14]: That was actually done previously, but hope it is now in a good place. 
https://github.com/justeat/fozzie-components/commit/ccac5a8e507c411629a3b0df8488e59bb263a7b6#diff-9ecd70013e8ffda2f1e9bcbf0fcc370f

**Review by DamianMullins — APPROVED** [review]

**Comment by zlatinivanov-je** [comment]
@ashleynolan The build keeps failing for the same reason even after reverting the change on [@vue/cli-plugin-eslint](https://github.je-labs.com/cogchord/ConsumerHelpWebApp/pull/146/commits/fbfccedbccb6482c020e99e203a579ff32226392). 
Could it be caching?

**Comment by ashleynolan** [comment]
@zlatinivanov-je I think it must be, as there's no other reason for this (especially if you can't recreate it locally).

I'd probably try opening this as a separate new PR and then hopefully the build should treat it as a whole new build. Not sure if there's a way of clearing this out otherwise.

**Comment by zlatinivanov-je** [comment]
Will re-open in attempt to clear CI cache

**Pull request closed without merging**